dbt/adapters/base/impl.py

```diff
@@ -1743,6 +1746,15 @@ def capabilities(cls) -> CapabilityDict:
     def supports(cls, capability: Capability) -> bool:
         return bool(cls.capabilities()[capability])
 
+    @available
```

Comment:
Does this really need to be available in the jinja context? I feel like we only need this for `dbt-core` to read.

Reply:
Ooop, nope, auto copy :p